Documentation/DataSources/PrivateWorkbooksDeprecation.md

```diff
@@ -0,0 +1,12 @@
+# Private Workbooks Deprecation
+
+Private Workbooks is going away by the end of 2020. You will still be able to access your private workbooks but any edit or save can only be saved as a Shared Workbook. 
+
+# Why? 
+There are several inconsistencies with the private workbook type that cause issues and confusion. 
+
+# How should you proceed?
+You can go to your private workbook and click "Save As" and select the "Shared Report" option from the drop down. If you don't want others to have access to your workbook, you can manually change the RBAC to deny access to others. If your private workbook is not migrated to a shared one by the time of deprecation, it may be too incompatible to migrate or even removed altogether. 
```

Comment:
not sure what `it may be too incompatible to migrate` will mean.  This probably would need to be rephrased to something like escalating to support to recover them, and we might need to make a jarvis action/etc to make that easier to do?

Reply:
removed that part since we're only removing ability to save new private workbooks at this time